<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Node_EF</class>
 <widget class="QWidget" name="Node_EF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Node_EF{
background-color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
}
QToolButton{
font: 20pt &quot;宋体&quot;;
color: rgb(0, 101, 150);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}
QToolButton:disabled{
font: 19pt &quot;宋体&quot;;
color: rgb(180, 180, 180);
border-color: rgb(180, 180, 180);
}
QGroupBox{
font: 15pt &quot;宋体&quot;;
color: rgb(255, 255, 255);
border:2px solid rgb(255, 255, 255);
border-radius:8px;
margin-top:20px;
}
QGroupBox:title{
subcontrol-origin: margin;
top:10px;
left: 10px;
}
QLabel{
font: 20pt &quot;宋体&quot;;
color: rgb(0, 122, 165);
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>455</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>功能操作</string>
   </property>
   <widget class="QToolButton" name="tBtnReset">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>40</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnClear">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>40</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnBack">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>40</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnCheck">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>40</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>校准</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBox_2">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>-10</y>
     <width>500</width>
     <height>480</height>
    </rect>
   </property>
   <property name="title">
    <string>工位-2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gBoxType">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>455</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>修改型号</string>
   </property>
   <widget class="QToolButton" name="tBtnSet">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>40</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>设定</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_T4">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>40</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_L8T4">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>40</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>L8T4</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_L12">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>40</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string> L12</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBox_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>-10</y>
     <width>500</width>
     <height>480</height>
    </rect>
   </property>
   <property name="title">
    <string>工位-1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
